docs/concepts/nix-expression-builds.md

```diff
@@ -137,7 +137,7 @@ If the latest version of a package isn't yet available in the Flox Catalog then
 ```nix title=".flox/pkgs/hello.nix"
 { hello, fetchurl }:
 
-hello.overrideAttrs (finalAttrs: old: {
+hello.overrideAttrs (finalAttrs: _oldAttrs: {
```

Comment:
> NB:
> 
> > If only a one-argument function is written, the argument has the
> > meaning of previousAttrs.
> 
> Maybe I missed it but I think we should clarify that otherwise from the outside it looks like an error sometimes passing a one arg callback and sometimes a two arg callback.

Can you be more specific about what you had in mind? A paragraph after the example that uses two arguments to explain the meaning and why it's different from the other examples?

Are variadic functions uncommon in Nix? It didn't strike me as particularly odd and was well documented upstream. Wondering if it's a convention thing or that people were used to the previous behaviour of `overrideAttrs`, which could influence what we write here.

Reply:
Merging to get this into the release - because I can't remember if the changes to the `preview` branch get locked in at the time of cutting the RC - but happy to follow up with another PR.